shared-module/_pixelbuf/PixelBuf.c

```diff
@@ -318,3 +331,23 @@ void common_hal__pixelbuf_pixelbuf_fill(mp_obj_t self_in, mp_obj_t fill_color) {
         common_hal__pixelbuf_pixelbuf_show(self_in);
     }
 }
+
+mp_int_t common_hal__pixelbuf_get_buffer(mp_obj_t self_in, mp_buffer_info_t *bufinfo, mp_uint_t flags) {
```

Comment:
Why is this added? I don't see it used anywhere.

Reply:
oops -- making a pixelbuf memoryview'able is a separate thing..